Frameworks/GLKit/GLKMath.mm

```diff
@@ -919,18 +949,20 @@ GLKIT_EXPORT GLKMatrix3 GLKMatrix3MakeZRotation(float rad) {
  @Status Interoperable
 */
 GLKIT_EXPORT GLKMatrix4 GLKMatrix4MakeRotation(float rad, float x, float y, float z) {
-    float magn = sqrtf(x * x + y * y + z * z);
+    const float magn = sqrtf(x * x + y * y + z * z);
+    GLKMatrix4 res = { 0 };
+    float invMagn;
+
     if (magn < COMPARISON_EPSILON) {
-        return GLKMatrix4MakeIdentity();
+        invMagn = -NAN;
+    } else {
+        invMagn = 1.f / magn;
     }
```

Comment:
...and anywhere else you think would benefit

---
In reply to: [66841122](https://github.com/Microsoft/WinObjC/pull/551#discussion_r66841122) [](ancestors = 66841122)